docs/resources/Channel.md

```diff
@@ -6,31 +6,38 @@ Represents a guild or DM channel within Discord.
+| recipients?            | array of [user](#DOCS_RESOURCES_USER/user-object) objects                  | the recipients of the DM                                                                                                                                                        |
+| icon?                  | ?string                                                                    | icon hash                                                                                                                                                                       |
+| owner_id?              | snowflake                                                                  | id of the DM or creator                                                                                                                                                         |
+| application_id?        | snowflake                                                                  | application id of the group DM creator if it is bot-created                                                                                                                     |
+| parent_id?             | ?snowflake                                                                 | for guild channels: id of the parent category for a channel (each parent category can contain up to 50 channels), for threads: id of the text channel this thread was created   |
+| last_pin_timestamp?    | ?ISO8601 timestamp                                                         | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                      |
+| rtc_region?            | ?string                                                                    | [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id for the voice channel, automatic when set to null                                                                  |
+| video_quality_mode?    | integer                                                                    | the camera [video quality mode](#DOCS_RESOURCES_CHANNEL/channel-object-video-quality-modes) of the voice channel, 1 when not present                                            |
+| message_count?         | integer                                                                    | an approximate count of messages in a thread, stops counting at 50                                                                                                              |
+| member_count?          | integer                                                                    | an approximate count of users in a thread, stops counting at 50                                                                                                                 |
```

Comment:
Yeah, responded to this elsewhere, but the first two are things that maybe could be valuable to future channel types, so figured they should be at the top level.  The others are _only_ ever valuable to a thread, so I decided to keep them in a "sub object".  `member` though definitely wouldn't go there, it's a per-user thing, so I don't think it belongs with per-channel settings, you know?

Reply:
I'm leaning towards yeeting thread_metadata and putting all those fields at the top level instead.  Thoughts?  Might be nice to be able to more easily tell that a field is thread-specific.

Reply:
I think I like the way it is currently, because some of the fields the thread-metadata are non-optional. Putting them on the top level means every parameter gets marked optional.